src/main/java/io/specto/hoverfly/junit/dsl/matchers/RequestMatcher.java

```diff
@@ -0,0 +1,14 @@
+package io.specto.hoverfly.junit.dsl.matchers;
+
+
+import io.specto.hoverfly.junit.core.model.FieldMatcher;
+
+public interface RequestMatcher {
+
+
+
+    FieldMatcher getFieldMatcher();
```

Comment:
I don't think this release will support multiple matches yet. Simply allowing an array of matchers won't work well. If you want exact matching, then you don't really need other matchers at all. 

I think this will be a separate issue, maybe have matcher builder like this: 
matchJsonpath().and().matchJsonpath()
or 
matchXml().and().matchXml()
or 
startsWith().and().contains() // chaining Glob matchers.

I don't see huge value for this now, we will do it incrementally.